doc/backends.md

```diff
@@ -1,52 +1,94 @@
-is covered, though.
+OpenQA (or actually os-autoinst) supports multiple backends to run the SUT. This
+document is meant to describe their particularities. So far only svirt and
+Vagrant are covered, though.
 
-For backend specific variables, there is a [separate documentation](backend_vars.asciidoc).
+For backend specific variables, there is a
+[separate documentation](backend_vars.asciidoc).
 
 ## svirt
 This backend establishes an SSH connection to another machine to start there
 a virtual machine using libvirt/virsh which uses QEMU, Xen, Hyper-V or VMWare
 under the hood.
 
+The svirt backend is enabled via `BACKEND=svirt` and the VM used under the hood
+is configured via `VIRSH_VMM_FAMILY`.
+
 ### Local setup
-Simply configure it to connect to the local machine and install libvirt/virsh.
+For running jobs using the svirt backend locally, follow the configuration steps
+explained in the following sub sections.
 
-#### Worker configuration
-Example configuration for using QEMU under the hood (add to `$OPENQA_CONFIG/workers.ini`):
+#### openQA worker configuration
+Example configuration for using the local libvirt setup with QEMU under the hood
+(add to `$OPENQA_CONFIG/workers.ini`):
 ```
 [2]
 BACKEND=svirt
-VIRSH_HOSTNAME=127.0.0.1 # use our own machine as svirt host
-VIRSH_USERNAME=root # see notes
+WORKER_CLASS=svirt,svirt-kvm
+VIRSH_HOSTNAME=127.0.0.1
+VIRSH_GUEST=127.0.0.1
+VIRSH_USERNAME=root
+VIRSH_PASSWORD=$THE_ROOT_PASSWORD
 VIRSH_CMDLINE=ifcfg=dhcp
 VIRSH_MAC=52:54:00:12:34:56
-VIRSH_OPENQA_BASEDIR=/var/lib # set in accordance with OPENQA_BASEDIR (by default /var/lib)
-WORKER_CLASS=svirt,svirt-kvm
+VIRSH_OPENQA_BASEDIR=/var/lib
 VIRSH_INSTANCE=1
-#VIRSH_PASSWORD=# see notes
-VIRSH_GUEST=127.0.0.1
 VIRSH_VMM_FAMILY=kvm
 VIRSH_VMM_TYPE=hvm
 ```
 
-##### Notes
-To allow an SSH connection to your local machine, either put your (root) password
-in the worker configuration or add your key to `$HOME/.ssh/authorized_keys`.
+If you add multiple instance, be sure to assign a different `VIRSH_INSTANCE` and
+`VIRSH_MAC`. For more details about the variables, checkout
+[backend_vars.asciidoc](backend_vars.asciidoc).
+
+`VIRSH_OPENQA_BASEDIR` must be set in accordance with `OPENQA_BASEDIR` if you
+changed that environment variable.
+
+Then start the worker slot as usual. If you invoke `isotovideo` directly
+(without openQA worker), these variables go to `vars.json`.
+
+#### SSH configuration
+To allow an SSH connection to your local machine, put your root password in the
+worker configuration as shown in the previous section. If your root password is
+entered from the configuration, that `type_string` command is logged. So be
+aware that your root password will end up in `autoinst-log.txt`.
+
+In any case, also be sure to allow root login in `/etc/ssh/sshd_config` and that
+`sshd` is actually started (e.g. using `zypper in openssh-server` and
+`systemctl start  sshd.service` under openSUSE).
```

Comment:
Remove the double space after "start" please. Also I recommend to leave out the `.service`